src/__tests__/homepage.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import {cleanup, render, screen} from '@testing-library/react';
+import HomePage from "../pages/Home/Home.page";
+import youtubeVideos from "../mocks/youtube-videos";
+
+describe('Home page Tests', () => {
+  afterEach(cleanup);
+
+  describe('Ensure elements get rendered', () => {
+    it('Title', () => {
+      render(<HomePage />);
+      expect(screen.getByText(/YouTube video search app/, {selector: 'h1'})).toBeInTheDocument();
+    });
+  });
+
+  describe('Ensure there the info from the mock is being shown', () => {
+    it("A random video's information will be displayed if it has videoid", () => {
+      render(<HomePage />);
+      const randVideoIdx = Math.floor((Math.random() * youtubeVideos.items.length));
+      const randomVideo = youtubeVideos.items[randVideoIdx];
+
+      if(randomVideo.id.videoId)
+        expect(screen.getByTestId(randomVideo.etag)).toBeInTheDocument();
+      else
+        expect(screen.getByTestId(randomVideo.etag)).not.toBeInTheDocument();
+    });
+
+    it('A video that does not have any description, must show channel title followed by a hypen', () => {
+      // render(<HomePage />);
```

Comment:
Unimplemented tests are better off from the commit.

Reply:
We can review that test to see how could it be best approached.